package.json

```diff
@@ -41,18 +43,17 @@
     "statsum": "^0.3.0",
```

Comment:
We should figure out why statsum is being included in here. With tc-lib-monitor being used I'm guessing it's no longer needed but I could be wrong.